Source/options.cpp

```diff
@@ -505,6 +513,13 @@ string_view OptionCategoryBase::GetDescription() const
 
 StartUpOptions::StartUpOptions()
     : OptionCategoryBase("StartUp", N_("Start Up"), N_("Start Up Settings"))
+    , gameMode("Game", OptionEntryFlags::NeedHellfireMpq | OptionEntryFlags::RecreateUI, N_("Game Mode"), N_("Play Diablo or Hellfire."), StartUpGameMode::Ask,
+          {
+              { StartUpGameMode::Diablo, N_("Diablo") },
+              // Ask is missing, cause we want to hide it from UI-Settings.
+              { StartUpGameMode::Hellfire, N_("Hellfire") },
+          })
+    , shareware("Shareware", OptionEntryFlags::NeedDiabloMpq | OptionEntryFlags::RecreateUI, N_("Restrict to Shareware"), N_("Makes the game compatible with the shareware. Allows to play multiplayer games with friends that don't own a diablo copy."), false)

```

Comment:
We started using the term "demo" for Spawn in the Android build so I think it's also appropriate here. Also, Diablo should be capitalized. Other than that, this is just a gentle suggestion to reword this description.

```suggestion
    , shareware("Shareware", OptionEntryFlags::NeedDiabloMpq | OptionEntryFlags::RecreateUI, N_("Restrict to Shareware"), N_("Makes the game compatible with the demo. Enables multiplayer with friends who don't own a full copy of Diablo."), false)
```

Reply:
The new description is a improvement, so changed the description,. 😉 

But your comment let me reflect how we should name the "not full version":
- `shareware`
- `spawn`
- `demo`

Currently we use all three. Do we want to use only one term?

`shareware` is used in the UI and (in this pr) in `ini` and some option code
`spawn` is used in mpq, code and arguments (`--spawn`)
`demo` is used in android build and description

Reply:
The Android build currently uses the phrase:
> Alternatively you can download the spawn version, to play the demo portion of the game.

So we use both `spawn` and `demo` in the same sentence. It seems our usage suggests that all three are valid terms for referring to that version of the game. This is largely up to personal opinion, but some brief notes about the three terms.

* Spawn is the term used in the file names for the exe and the mpq. It is perhaps the most accurate term to use for describing this version of the game to those who are familiar with it. However, it's something of a made-up term and doesn't really convey much useful information to anyone who doesn't already know what it is.
* Shareware is perhaps the best term to convey the idea that this version of the game is free to play. It typically refers to closed-source-but-freely-available software so it may not be all that accurate to use it in the context of DevilutionX. However, Blizzard did use this term in the background art for the main menu so it's probably here to stay. This term may not be as recognizable to players who are not as familiar with personal computers.
* Demo is perhaps the best term to convey the idea that you will only be experiencing a subset of the full game's content. I would say this is the most broadly understood of the three terms since it's used in a variety of contexts.

I think it's fairly safe to default to `demo` in most contexts if you're having trouble figuring out which one to use.